README.rst

```diff
@@ -0,0 +1,81 @@
+SaltyRTC Signalling Server
+==========================
+
+|Travis| |codecov| |PyPI|
+
+This is a SaltyRTC server implementation for Python 3.4+ using
+`asyncio`_.
+
+Note
+****
+
+On machines where Python 3 is not the default Python runtime, you should
+use ``pip3`` instead of ``pip``.
+
+Prerequisites
+*************
+
+.. code-block:: bash
+
+    $ sudo apt-get install python3 python3-pip
+
+We recommend using `venv`_ to create an isolated Python environment:
+
+.. code-block:: bash
+
+    $ pyvenv venv
+
+You can switch into the created virtual environment *venv* by running
+this command:
+
+.. code-block:: bash
+
+    $ source venv/bin/activate
+
+While the virtual environment is active, all packages installed using
+``pip`` will be installed into this environment.
+
+To deactivate the virtual environment, just run:
+
+.. code-block:: bash
+
+    $ deactivate
+
+If you want easier handling of your virtualenvs, you might also want to
+take a look at `virtualenvwrapper`_.
+
+Installation
+************
+
+If you are using a virtual environment, activate it first.
+
+Install the module by running:
+
+.. code-block:: bash
+
+    $ pip install saltyrtc
```

Comment:
It could be either `saltyrtc-server` and the common parts will be moved into another package. I don't like that idea. Or both server and client will be in this package. Yes, that would require some more dependencies at some point, but I think that should be okay.